script.js

```diff
@@ -1,12 +1,60 @@
 //You can edit ALL of the code here
+
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  // console.log(getAllEpisodes());
 }
 
+let container = document.querySelector('#root');
+let searchBar = document.querySelector("input");
+
+  // console.log(searchBar + "is my searchBar");
+  // console.log(container + "is my holder");
+
+  
+  document.querySelector("input").setAttribute("type", "text");
+  document.querySelector("input").setAttribute("id", "search");
+  document.querySelector("input").setAttribute("placeholder", "search for Episode");
+  document.getElementById("search").style.zIndex = "1";
+  document.getElementById("search").style.position = "absolute";
+  document.getElementById("search").style.margin = "auto";
+  document.getElementById("search").style.padding = "auto";
+  
+  // console.log(searchBar + "this is my updated searchbar");
+
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+let output = "";
+  for (let value of episodeList) {
```

Comment:
Line 28, 32 to 35. Demonstrating very good understanding and skills of retreiving data from a variable(episodeList) and assigning them to the appropriate variables to display episode titles etc. Keep on practising and building on these concepts as well as their scopes